packages/@aws-cdk/aws-events-targets/package.json

```diff
@@ -102,8 +101,12 @@
     "@aws-cdk/aws-stepfunctions": "0.0.0",
     "@aws-cdk/core": "0.0.0",
     "@aws-cdk/custom-resources": "0.0.0",
-    "constructs": "^3.2.0"
+    "constructs": "^3.2.0",
+    "aws-sdk": "^2.848.0"
```

Comment:
I'm not too much of a fan of this.

There are probably just data files we could be copying out of this dependency at build time, no?

In fact, I'm fairly sure there are other modules that are already doing the same. I think probably the `AwsCustomResource`, which does almost the same as this construct does ?

Reply:
Got it.